integration-test/src/it/groovy-manipulator-file/verify.groovy

```diff
@@ -30,7 +30,7 @@ assert !jsonString.contains ("resolved:")
 def buildLog = new File( basedir, 'build.log' )
 def message = 0
 buildLog.eachLine {
-   if (it.contains( "groovyScripts file:///")) {
+   if (it.contains( "groovyScripts file:")) {
```

Comment:
Why this change?

Reply:
Because if `${basedir}` does not begin with a `/` then `file://${basedir}` won't have three slashes and this check will fail, so it should be sufficient to check for `file:` which is what seems to be used everywhere else in the code anyway.

Reply:
Ok